dbmongo/js/compact/applyPatchesToMemory.ts

```diff
@@ -0,0 +1,24 @@
+import "../globals"
+
+type DataType = string // TODO: use BatchDataType instead
+
+export function applyPatchesToMemory(
+  hashToAdd: Record<DataType, Set<DataHash>>,
+  hashToDelete: Record<DataType, Set<DataHash>>,
+  memory: CurrentDataState
+): void {
+  // On retire les cles restantes de la memoire.
```

Comment:
```suggestion
  // Prise en compte des suppressions de clés dans la mémoire
```

("clés restantes" n'a plus de sens dans ce contexte).